src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("Search Results", () => {
+    const validProps = {
+        results: [
+            {
+                key: [123],
+                className: "search-results__images",
+                src: "StubImage1",
+                alt: "search-results__image",
+            },
+
+            {
+                key: [124],
+                className: "search-results__images",
+                src: "StubImage2",
+                alt: "search-results__image",
+            },
+
+            {
+                key: [125],
+                className: "search-results__images",
+                src: "StubImage3",
+                alt: "search-results__image",
+            }
+        ]
+    };
+    const { asFragment } = render(
+        <SearchResults
+            results={validProps.results}
+        />);
+
+    render(
+        <SearchResults
+            results={validProps.results}
+        />
+    );
+    it("renders correctly", () => {
+        expect(asFragment()).toMatchSnapshot()
+    })
```

Comment:
Theres a bit of unnecessary noise here. The `const { asFragment } = render` part should be in the `it` block, and the render block is not needed:
```
it("renders correctly", () => {
    const { asFragment } = render(<SearchResults results={validProps.results} />);

    expect(asFragment()).toMatchSnapshot();
  });
```